### ISSUE TITLE
Investigate why the comment was not posted for #71

### ISSUE THREAD

**Issue by tuliren** [stoat-dev/stoat-action#72]
It's likely because the PR was created from a fork.

**Comment by jrhizor** [comment]
Yeah it's just because it's for a fork. 

Maybe we should allow manually triggering for a whitelisted commit on a fork somehow? The build was triggered when you reviewed, right? Those aren't differentiated in our db vs non-fork PRs, but that's probably fine since it's approved?

**Comment by tuliren** [comment]
> The build was triggered when you reviewed, right?

Yes, I approved the action on that PR, and the action did run. It's probably a permission issue from the GITHUB_TOKEN.

**Comment by jrhizor** [comment]
All of the `GITHUB_TOKEN`-related auth in the action run looks like it worked properly, it's the webhook that wasn't triggered or allowed (not sure)
